anx/provider/loadbalancer/reconciliation/reconciliation_test.go

```diff
@@ -42,6 +44,8 @@ var _ = Describe("reconcile", func() {
 
 		svcUID = rand.String(32)
 
+		retriever = newStateRetriever(context.TODO(), apiClient, fmt.Sprintf("anxccm-svc-uid=%v", svcUID), []string{testLoadBalancerIdentifier})
```

Comment:
let's extract the tag into a variable to use here and in the other places - instead of the current `svcUID` variable?